src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig

```diff
@@ -82,6 +82,8 @@
 
     <h2>Translation Messages</h2>
 
+    {% block before_translation_messages %}{% endblock %}
```

Comment:
Instead of `before`/`after` blocks, I would prefer one block that wraps all the content, which can then be overridden easily with:

```
{% block contents %}
  before
  {{ parent() }}
  after
{% endblock %}
```